doc/man3/OSSL_STORE_LOADER.pod

```diff
@@ -106,9 +113,14 @@ initialized, to create a private data store (B<OSSL_STORE_LOADER_CTX>, see
 above), and to return it.
 If something goes wrong, this function is expected to return NULL.
 
-=item B<OSSL_STORE_open_fn>
+B<OSSL_STORE_open_with_libctx_fn> does the same thing as B<OSSL_STORE_open_fn>,
```

Comment:
matt pointed out that we use the OSSL_STORE_open_with_libctx_fn one as the main description so that this one should be B<OSSL_STORE_open_fn> does the same thing as OSSL_STORE_open_with_libctx_fn but uses NULL for X & Y.

Reply:
It's really just an implementation detail, so I guess this is more about what we want to promote.